lib/salemove/process_handler/pivot_process.rb

```diff
@@ -170,10 +170,14 @@ def handle_request(input)
           handle_exception(exception, input)
         end
 
+        PROCESSED_REQUEST_LOG_KEYS = [:error, :success]
+
         def delegate_to_service(input)
           request_id = input[:request_id]
           result = PivotProcess.benchmark(input) { @service.call(input) }
-          PivotProcess.logger.info "Processed request", result.merge(request_id: request_id)
+          PivotProcess.logger.info(
+            "Processed request", result.select{|k, _| PROCESSED_REQUEST_LOG_KEYS.include?(k)}.merge(request_id: request_id)
+          )
```

Comment:
nice way to split these kind of lines is
```
          PivotProcess.logger.info "Processed request",
            result.select{|k, _| PROCESSED_REQUEST_LOG_KEYS.include?(k)}.merge(request_id: request_id)
```
No parantheses, less width, main thing in in the first line, attributes on the second